olive/common/ort_inference.py

```diff
@@ -26,9 +26,12 @@ def get_ort_inference_session(
     session_options = inference_settings.get("session_options", {})
     inter_op_num_threads = session_options.get("inter_op_num_threads")
     intra_op_num_threads = session_options.get("intra_op_num_threads")
+    enable_profiling = session_options.get("enable_profiling", False)
```

Comment:
Should we add this config into one example which is running in our CI pipeline?

Reply:
Yeah, I already add the config in rwkv example. That's the reason I created the PR.